Publish OSLC PROMCODE OS files under OSLC NS

tomkamimura opened the pull request:
I have modified promcode-shapes.ttl and promcode-vocab.ttl files per the comments of Andrew and Jim.

berezovskyi commented on content/ns/promcode/promcode-shapes.ttl:
```diff
@@ -35,7 +35,7 @@
   dcterms:source <https://tools.oasis-open.org/version-control/browse/wsvn/oslc-promcode/shape/trunk/promcode-shapes.ttl> ;
   dcterms:isPartOf <https://tools.oasis-open.org/version-control/browse/wsvn/oslc-promcode/shape/trunk/spec.html> ;
-  dcterms:hasVersion "CS02" .
+  dcterms:hasVersion "CS03" .
```
Why CS03?

tomkamimura replied:
Sorry, it should be CS02. The latest version has corrected it.

tomkamimura commented:
Andrew,

Sorry, it is CS02. I will push the changed files.

Tom

> 2022/02/04 16:10、Andrew Berezovskyi ***@***.***>のメール:
> 
> 
> @berezovskyi commented on this pull request.
> 
> In content/ns/promcode/promcode-shapes.ttl <https://github.com/oslc-op/website/pull/307#discussion_r799213015>:
> 
> > @@ -35,7 +35,7 @@
>    dcterms:license <http://www.apache.org/licenses/LICENSE-2.0> ;
>    dcterms:source <https://tools.oasis-open.org/version-control/browse/wsvn/oslc-promcode/shape/trunk/promcode-shapes.ttl> ;
>    dcterms:isPartOf <https://tools.oasis-open.org/version-control/browse/wsvn/oslc-promcode/shape/trunk/spec.html> ;
> -  dcterms:hasVersion "CS02" .
> +  dcterms:hasVersion "CS03" .
> Why CS03?
> 
> —
> Reply to this email directly, view it on GitHub <https://github.com/oslc-op/website/pull/307#pullrequestreview-872733686>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AXJDHIXJJCO6PKQETHRFUG3UZN3WXANCNFSM5NQUP2BQ>.
> Triage notifications on the go with GitHub Mobile for iOS <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>. 
> You are receiving this because you authored the thread.
>

berezovskyi commented:
I have provided my latest comment here: https://github.com/oslc-op/website/issues/278#issuecomment-1043113242

tomkamimura commented:
promcode-shapes.ttl and promcode.vocab.ttl files are updated on February 24, 2022. See the comment at https://github.com/oslc-op/website/issues/278

tomkamimura commented:
Andrew,

I posted two comments into #278 at https://github.com/oslc-op/website/issues/278 <https://github.com/oslc-op/website/issues/278> . Please take a look.

Tom Kamimura



> 2022/02/18 0:57、Andrew Berezovskyi ***@***.***>のメール:
> 
> 
> I have provided my latest comment here: #278 (comment) <https://github.com/oslc-op/website/issues/278#issuecomment-1043113242>
> —
> Reply to this email directly, view it on GitHub <https://github.com/oslc-op/website/pull/307#issuecomment-1043114267>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AXJDHIQCYKU3IBMDSIPY7ZDU3ULIDANCNFSM5NQUP2BQ>.
> Triage notifications on the go with GitHub Mobile for iOS <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>. 
> You are receiving this because you authored the thread.
>

tomkamimura commented:
I ran ShapeChecker again on the lastest promcode-shapes.ttl and promcode-voca.ttl today (March 2, 2022). One warning generated which says that shapes.ttl does not refernce voca.ttl. Here is the result.

Results from ShapeChecker run on Wed Mar 02 15:16:52 JST 2022

Command line arguments:
-t Warning -C -q EmbeddedWhitespace -q MissingPeriod -x http://open-services.net/ns/promcode -x
http://open-services.net/ns/promcode#.* -x http://purl.org/dc/terms/ -x http://purl.org/dc/terms/.* -v
/Users/tomkamimura/PROMCODE/oslc-promcode/PROMCODE/turtle_files/promcode-vocab.ttl -s
../../../turtle_files/promcode-shapes.ttl

A total of 1 issue was found (0 info, 1 warnings, 0 errors)

Checked vocabulary file:/Users/tomkamimura/PROMCODE/oslc-promcode/PROMCODE/turtle_files/promcode-vocab.ttl from file:/Users/tomkamimura/PROMCODE/oslc-promcode/PROMCODE/turtle_files/promcode-vocab.ttl, with 0 issues (0 info, 0 warnings, 0 errors)

Checked shapes from file:/Users/tomkamimura/PROMCODE/oslc-promcode/PROMCODE/build/install/ShapeChecker/../../../turtle_files/promcode-shapes.ttl, with 0 issues (0 info, 0 warnings, 0 errors)

Warning: This vocabulary was given, but not referenced in the given shapes:
file:/Users/tomkamimura/PROMCODE/oslc-promcode/PROMCODE/turtle_files/promcode-vocab.ttl

tomkamimura commented:
We looked at our turtle files and our script to run ShapeCheck. There were some blank characters in places that may have caused a warning. Also, we changed parameters of the script to detect such blank characters. After fixing those blank characters, we ran the script and a warning is now gone. Here is the output of ShapeCheck. So, I think we are good now. I have modified the turtle files in the active PQ. @berezovskyi, will you please move ahead to get an agreement from oslc-ops?

Command line arguments:
-t Warning -C -q unusedVocabulary -x http://open-services.net/ns/promcode -x http://open-services.net/ns/promcode#.* -x
http://purl.org/dc/terms/ -x http://purl.org/dc/terms/.* -v
/Users/tomkamimura/PROMCODE/oslc-promcode/PROMCODE/turtle_files/promcode-vocab.ttl -s
../../../turtle_files/promcode-shapes.ttl

A total of 0 issues were found (0 info, 0 warnings, 0 errors)

Checked vocabulary file:/Users/tomkamimura/PROMCODE/oslc-promcode/PROMCODE/turtle_files/promcode-vocab.ttl from file:/Users/tomkamimura/PROMCODE/oslc-promcode/PROMCODE/turtle_files/promcode-vocab.ttl, with 0 issues (0 info, 0 warnings, 0 errors)

Checked shapes from file:/Users/tomkamimura/PROMCODE/oslc-promcode/PROMCODE/build/install/ShapeChecker/../../../turtle_files/promcode-shapes.ttl, with 0 issues (0 info, 0 warnings, 0 errors)

tomkamimura commented:
One more comment. We double checked our turtle files and confirmed the all the terms defined in vocab.ttl are referenced in shapes.ttl. You might think that a warning is gone because we added the parameter -q unusedVocabulary. We tested turtle files of QM that does not produce any warning. The script of Qm uses this parameter also. But, when we removed this parameter in the script of QM, it produced the same warning as the one we used to get. So, I don't know what causes a warning when the parameter unusedVocabulary is not used. At least, we can say that if there is a problem, QM probably has the same problem. In any case, the warning message produced by ShapeCheck does not seem accurate.

berezovskyi commented:
@tomkamimura getting ready to merge and publish. Could you please check the files again, please? I saw on the lists that PROMCODE hit the OASIS Standard stage, but the files still say CS03. Should we change those to OS before publication?

tomkamimura commented:
Andrew,

Thank you for your help. Yes, PROMCODE is now an OASIS standard, and that means that editorial change will be needed. I will check the files and will make necessary change before they are merged.

Tom Kamimura

> 2022/04/07 21:53、Andrew Berezovskyi ***@***.***>のメール:
> 
> 
> @tomkamimura <https://github.com/tomkamimura> getting ready to merge and publish. Could you please check the files again, please? I saw on the lists that PROMCODE hit the OASIS Standard stage, but the files still say CS03. Should we change those to OS before publication?
> 
> —
> Reply to this email directly, view it on GitHub <https://github.com/oslc-op/website/pull/307#issuecomment-1091698732>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AXJDHIRBLXLYS44ENTVKDCTVD3LDNANCNFSM5NQUP2BQ>.
> You are receiving this because you were mentioned.
>

tomkamimura commented:
Andrew, I have updated shapes.ttl and vocab.ttl files that rerflect the OASIS Standard version.

tomkamimura commented:
@berezovskyi , the correct date is March 11, 2022. I changed the wrong date in some files to 2022-03-11.
